conda_smithy/configure_feedstock.py

```diff
@@ -591,6 +591,13 @@ def _render_ci_provider(
             f"{platform}_{arch}"
         ].replace("_", "-")
 
+        # set the environment variable for OS version
+        # currently we only care about cos6/cos7 for linux64, but it might be extended in the future
```

Comment:
Can we just do this for all linux archs instead of just `linux64`?

Reply:
I have no opinion here, but do we currently need to worry about other linux archs? I might have missed something but haven't seen such a case so far.